test/system/104-run.bats

```diff
@@ -166,3 +166,14 @@ teardown() {
   assert_line --index 2 "Error: failed to invoke command /etc in container $(get_latest_container_name)"
   assert [ ${#lines[@]} -eq 3 ]
 }
+
+@test "run: Try to run a Toolbx command from a container (check error message propagation)" {
+  create_default_container
+
+  run $TOOLBOX run toolbox foo
```

Comment:
... not this test case, because it doesn't demonstrate the problem that the above code change tries to address.

When trying to run an invalid command line, such as with an unknown command, the error is returned very early in the lifetime of the `toolbox(1)`  binary before anything has been forwarded over `flatpak-spawn(1)`.  Specifically, within the `rootCmd.Execute()` call.

Instead, we need something that happens after the CLI invocation has been forwarded over `flatpak-spawn(1)`.  Such as, within a command's `RunE` function.  I think `toolbox create --distro unknown --release invalid` can work.

I would still keep this test case, because it still tests *some* thing, just not *this* thing.  :)

Reply:
> I would still keep this test case, because it still tests some thing, just not this thing. :)

I tried to resurrect just the test to prove my theory in https://github.com/containers/toolbox/pull/1525